### DISCUSSION TITLE
Reactor instant meltdown after loading Outpost

### DISCUSSION THREAD

**Discussion by zeroinformatique** [FakeFishGames/Barotrauma#13964] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

I'm the host, we use the Orca 2.
Just before docking, during the mission, I was handling the Reactor manually: it was unstable at this moment.
We dock, get the loading screen, then instantly a Meltdown alarm and 2 seconds later the Reactor explodes.
It was unavoidable.

Isn't the reactor supposed to switch to automatic mode on docking, to avoid meltdowns?

### Reproduction steps

1. Start a mission
2. Handle the reactor manually before docking, using two fuel rods so it heats up quickly
3. Dock while the reactor was slightly misconfigured (heating fast but still okay during docking)
4. Meltdown occurs almost instantly after loading

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
I think switching it to auto on docking would be annoying for those guys with reactor automation circuits

You really shouldn't have touched it before docking
For same reason you should swap o2 tanks or take off your suit before docking or you might spawn suffocated

Just repair the damage, live with the consequences, never touch that before docking again

**Comment by zeroinformatique** [comment]
> I think switching it to auto on docking would be annoying for those guys with reactor automation circuits

Okay, then the game can enable automation only if no such input is connected.

The issue is caused by the lag at the start of the game, and the teleportation inside the outpost. There are situations which could be avoided if the simulation delayed the reactor simulation in a outpost (it's not really used anyway). Here it exploded in two seconds, it was impossible to avoid.

> You really shouldn't have touched it before docking

It was obligatory, one of the rod expired and we were flooding. I didn't have time to calibrate the reactor when we docked urgently. It was actually decently stable and was super surprised it exploded so quickly. We're only two players, one captain and me running everywhere to maintain the sub afloat. You can't ignore that some situations require to act quick; still the reactor should not have exploded in this situation.

> For same reason you should swap o2 tanks or take off your suit before docking or you might spawn suffocated

The game should definitely delay the simulation until everyone is connected at the beginning of a round, especially in a outpost. This is another issue which makes no sense.

> Just repair the damage, live with the consequences, never touch that before docking again

Enabling the auto-reactor on mission start if no automation wire is connected totally fixes the issue.
